packages/shylu/shylu_dd/frosch/test/Thyra_Xpetra_Elasticity/ParameterLists/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ DEST_FILES ParameterList_GDSWPreconditioner_ILU.xml
            ParameterList_TwoLevelPreconditioner_IPOUHarmonic_RGDSW_Klu.xml
            ParameterList_TwoLevelPreconditioner_RGDSW_ILU.xml
            ParameterList_TwoLevelPreconditioner_RGDSW_Klu.xml
+           ParameterList_TwoLevelPreconditioner_GDSW_ML.xml
+           ParameterList_TwoLevelPreconditioner_GDSW_ML3.xml
+           ParameterList_TwoLevelPreconditioner_RGDSW_ML.xml
+           ParameterList_TwoLevelPreconditioner_RGDSW_ML3.xml
```

Comment:
Do we always need a different parameter list when going from 2D to 3D? This was not the case before.

Reply:
The TwoLevelPreconditioner needs the dimension. The CoarseOperator could pass the value to the `CoarseSolver_` in that way, we do not have to use different parameter lists.